misc/ApplicationHostTestProjects/src/SyntaxException/program.cs

```diff
@@ -0,0 +1,7 @@
+﻿class Program
+{
+	public static void Main()
+	{
+		System.Console1.WriteLine("Test");
```

Comment:
Using a type name that doesn't exist is not a syntax error. Either rename the project or add an actual syntax error